### PR TITLE
Always set request_id

### PR DESCRIPTION
**Pull request by ChrisBr**
We will install a `before_action` which will always set the `request_id`.

### CODE REVIEW

File: spec/support/rails4/app.rb [hunk 1]
```diff
@@ -32,6 +32,10 @@ class ApplicationController < ActionController::Base; end
 class WidgetsController < ApplicationController
   prepend_view_path File.join(__dir__, "..", "views")
 
+  before_action do
+    InfluxDB::Rails.current.values = { key: :value }
```
Comment (hennevogel) [inline spec/support/rails4/app.rb:36]: Can you use more speaking names? Like `{ user: 'chris' }`

File: lib/influxdb/rails/railtie.rb [hunk 2]
```diff
@@ -23,6 +23,10 @@ class Railtie < ::Rails::Railtie # :nodoc:
           include InfluxDB::Rails::AirTrafficController
           require "influxdb/rails/instrumentation"
           include InfluxDB::Rails::Instrumentation
+
+          before_action do
```
Comment (hennevogel) [inline lib/influxdb/rails/railtie.rb:27]: Too easy...

File: spec/integration/metrics_spec.rb [hunk 3]
```diff
@@ -1,16 +1,26 @@
...
   describe "in a normal request" do
     it "should result in attempts to write metrics via the client" do
       expect(InfluxDB::Rails.client).to receive(:write_point).exactly(5).times
-      get :index
+      get "/widgets"
+    end
+
+    context "additional values" do
+      it "should result in attempts to write metrics via the client" do
+        allow_any_instance_of(ActionDispatch::Request).to receive(:request_id).and_return(:request_id)
```
Comment (hennevogel) [inline spec/integration/metrics_spec.rb:16]: Can you use more speaking names? Like `allow_any_instance_of(ActionDispatch::Request).to receive(:request_id).and_return('12345')`

**Review by hennevogel — APPROVED** [review]

**Comment by ChrisBr** [comment]
Actually, I prefer to have descriptive (`key: value`) over realistic (`user: :chris`) test data 🤔

**Comment by hennevogel** [comment]
> I prefer to have descriptive over realistic test data

You're the boss...

**Pull request merged**